azure-pipelines-official.yml

```diff
@@ -23,109 +24,109 @@ trigger:
-            condition: always()
-
-          - task: AzureCLI@2
-            displayName: 'Push SlnGen.Corext'
-            inputs:
-              azureSubscription: 'CloudBuild-Push-v2'
-              scriptType: 'pscore'
-              scriptLocation: 'inlineScript'
-              inlineScript: |
-                # Get an access token for Azure DevOp (resource id is for AzDO)
-                $accessToken = az account get-access-token --query accessToken --resource 499b84ac-1321-427f-aa17-267ca6975798 -o tsv
-                
-                # Set the access token as a secret, so it doesn't get leaked in the logs
-                Write-Host "##vso[task.setsecret]$accessToken"
-                
-                # Override the apitoken of the nuget service connection, for the duration of this stage
-                # Service connection is for "microsoft.slngen Official"
-                Write-Host "##vso[task.setendpoint id=d8767bc1-a109-4cb7-80c2-6ac1ebf4346e;field=authParameter;key=apitoken]$accessToken"
-
-          - task: NuGetAuthenticate@1
-            inputs:
-              forceReinstallCredentialProvider: true
-              nuGetServiceConnections: 'CloudBuild-Push'
-
-          - task: NuGetCommand@2
-            displayName: 'Push SlnGen.Corext'
-            inputs:
-              command: 'push'
-              packagesToPush: '$(Pipeline.Workspace)/$(ArtifactsDirectoryName)/**/SlnGen.Corext*.nupkg'
-              nuGetFeedType: 'external'
-              publishFeedCredentials: 'CloudBuild-Push'
+      name: VSEngSS-MicroBuild2022-1ES
+      demands:
+      - msbuild
+      - visualstudio
+      os: windows
+    stages:
+    - stage: ''
+      displayName: 'Build'
+      jobs:
+      - job: Build
+        displayName: 'Build'
+        pool:
+          name: 'VSEngSS-MicroBuild2022-1ES'
+        templateContext:
+          mb:
+            signing:
+              enabled: true
+              signType: 'real'
+              zipSources: false
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish Artifacts'
             condition: always()
+            targetPath: '$(ArtifactsDirectoryName)'
+            artifactName: $(ArtifactsDirectoryName)
+        steps:
+        - script: 'echo ##vso[task.setvariable variable=SignType;]Real'
+          displayName: 'Set SignType to Real for tagged commits'
+          condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/v'))
+        - task: UseDotNet@2
+          displayName: 'Install .NET $(DotNet6Version)'
+          inputs:
+            version: '$(DotNet6Version)'
+        - task: UseDotNet@2
+          displayName: 'Install .NET $(DotNet7Version)'
+          inputs:
+            version: '$(DotNet7Version)'
+        - task: UseDotNet@2
+          displayName: 'Install .NET $(DotNet8Version)'
+          inputs:
+            version: '$(DotNet8Version)'
+            includePreviewVersions: true
+        - script: |
```

Comment:
You can clean this up if you want now that .NET 9 is installable through the UseDotNet task.  It used to only be possible to install .NET 9 with the install scripts but I never got around to this.

Reply:
yeah for sure going to clean things up, remove net7, etc. Once I get the pipeline to actually run :)